build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -233,37 +233,33 @@ def convert_target(target):
   if target.startswith("//llvm-external-projects/iree-dialects"):
     return _convert_iree_dialects_target(target)
 
+  if target.startswith("@"):
+    raise KeyError(f"No conversion found for target '{target}'")
+
   # IREE root paths map to package names based on explicit rules.
```

Comment:
Update the comment to explain the rules?

Reply:
Added some comments